dashboard/src/main/java/org/openedx/courses/presentation/DashboardGalleryView.kt

```diff
@@ -728,7 +736,7 @@ private fun PrimaryCourseButtons(
                         if (futureAssignments.size == 1) {
                             resumeBlockId(primaryCourse, nearestAssignment.blockId)
```

Comment:
This case needs to be handled for both past and future cases, as the action sent in this instance, 'resume_course,' is incorrect.

Reply:
addressd.